cmd/restic/cmd_backup.go

```diff
@@ -156,11 +152,14 @@ func filterExisting(items []string) (result []string, err error) {
 	return
 }
 
-// readFromFile will read all lines from the given filename and return them as
-// a string array, if filename is empty readFromFile returns and empty string
-// array. If filename is a dash (-), readFromFile will read the lines from the
+// readPatternsFromFile will read all lines from the given filename and return
+// them as a string slice, after stripping off leading and trailing whitespace
+// and removing empty lines and comments.
```

Comment:
Comment is a bit outdated due to the changes to whitespace, suggest:

~~~
// readPatternsFromFile will read all lines from the given filename and return
// them as a string slice, after removing empty lines and possibly comments.
~~~

Reply:
I just realized that lines starting with whitespace, then a '#' are no longer comments. I fixed this by renaming this function readLines and restricting its functionality.